test/extended/images/s2i_ruby.go

```diff
@@ -0,0 +1,76 @@
+package images
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("images: s2i: ruby", func() {
+	defer g.GinkgoRecover()
+	var (
+		railsTemplate = "https://raw.githubusercontent.com/openshift/rails-ex/master/openshift/templates/rails-postgresql.json"
+		oc            = exutil.NewCLI("s2i-php", exutil.KubeConfigPath())
+		modifyCommand = []string{"sed", "-ie", `s%render :file => 'public/index.html'%%`, "app/controllers/welcome_controller.rb"}
+		removeCommand = []string{"rm", "-f", "public/index.html"}
```

Comment:
Cause in rails the public/index.html is served by default when requiring the app root. You have to delete it so other views can get served.